File: java/URI_1036.java
package URI;
import java.util.Locale;
import java.util.Scanner;

public class URI_1036 {

	public static void main(String[] args) {
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		double A, B, C, DELTA, X1, X2;
		
		A = sc.nextDouble();
		B = sc.nextDouble();
		C = sc.nextDouble();
		
		DELTA = B * B - 4 * A * C;
		if(DELTA < 0 || A == 0) {
			System.out.println("Impossivel calcular");
		}
		else {
			X1 = (- B + Math.sqrt(DELTA)) / (2 * A);
			X2 = (- B - Math.sqrt(DELTA)) / (2 * A);
			System.out.printf("R1 = %.5f%n", X1);
			System.out.printf("R2 = %.5f%n", X2);			
		}	
		
		sc.close();
	}
}

File: kotlin/URI_1036.kt
import java.util.*
import kotlin.math.sqrt

fun main(args: Array<String>) {
    Locale.setDefault(Locale.US)
    val sc = Scanner(System.`in`)

    val a = sc.nextDouble()
    val b = sc.nextDouble()
    val c = sc.nextDouble()

    val delta = b * b - 4 * a * c
    when {
        delta < 0 || a == 0.0 -> println("Impossivel calcular")
        else -> {
            val x1 = (-b + sqrt(delta))/ (2 * a)
            val x2 = (-b - sqrt(delta))/ (2 * a)
            println("R1 = ${"%.5f".format(x1)}")
            println("R2 = ${"%.5f".format(x2)}")
        }
    }
}
